src/components/Footer/Footer.jsx

```diff
@@ -114,6 +114,11 @@ const Footer = () => {
                       <ListItemText primary="Privacy and Policy" />
                     </FooterLink>
                   </ListItem>
+                  <ListItem>
+                    <FooterLink to="/community">
+                      <ListItemText primary="Community Page" />
```

Comment:
Just keep the text as "Community" no need "Page". Also the header appears empty

Reply:
@everly-gif I think there is some problem in navbar, in the website also it is not appearing properly pls check it and this happened locally also after I pulled the latest changes.

Reply:
Check this - 
https://github.com/user-attachments/assets/58922679-e1c8-4a90-9488-516ccc0926fb